workloads/network-perf-v2/run.sh

```diff
@@ -15,9 +15,20 @@ log "###############################################"
 log "Workload: ${WORKLOAD}"
 log "###############################################"
 
-timeout $TEST_TIMEOUT ./k8s-netperf --debug --metrics --all --config ${WORKLOAD} --search $ES_SERVER --tcp-tolerance ${TOLERANCE} --clean=true
-run=$?
-
+echo $WORKLOAD
+        if [ $WORKLOAD == iperf ];then
```

Comment:
$WORKLOAD is the config file, it is required -- without it, k8s-netperf will run the default workload.